packages/locale/pt.yml

```
@@ -26,28 +26,25 @@ why-it-works:
     de aprendizado seja moldado de acordo com as suas necessidades.
   teamwork.title: Trabalho em equipe
   teamwork.description: >
-    Participe das mentorias personalizadas individuais e projetos
-    em grupo. É você quem escolhe como contribuir com a comunidade.
+    Participe das mentorias de trilhas e projetos em grupo. É você quem escolhe como contribuir com a comunidade.
 
 mentoring:
   title: Nossa metodologia
   description: >
     É estruturada por mentorias que são direcionadas por trilhas
     conforme a necessidade e o tempo de cada pessoa, focando no acesso
     ao mercado de trabalho.
-  personalized-mentoring.title: Mentoria personalizada
+  personalized-mentoring.title: Mentoria de trilha
   personalized-mentoring.description: >
-    A mentoria personalizada tem como objetivo um acompanhamento individual
-    do mentorando, focado em trilhas introdutórias e exercícios especializados.
-  pair-mentoring.title: Mentoria em dupla
+    A mentoria de trilha tem como objetivo guiar o mentorando nos seus estudos, focado em trilhas introdutórias e exercícios especializados.
+  pair-mentoring.title: Mentoria de projetos
   pair-mentoring.description: >
-    A mentoria em dupla tem como objetivo a troca de experiência entre mais de um
-    mentorado, pair programming e colaboração no desenvolvimento de exercícios e projetos.
-  project-mentoring.title: Mentoria de projetos
-  project-mentoring.description: >
     É focada em desenvolver projetos internos ou externos. O mentorando vivencia trabalho
     em equipe, pair programmings e code reviews, todos estão 100% do tempo simulando o dia
```

vi aq q nem foi vc q escreveu essa parte mas acho que cabe uma alteraçao 
```suggestion
    em equipe, pair programmings e code reviews, com o objetivo principal de simular o dia
```